examples/set-annotations/imperative/README.md

```diff
@@ -0,0 +1,29 @@
+# set-annotations: Imperative Example
+
+### Overview
```

Comment:
nit: These should be h2 (##).

Reply:
Does this comment conflict with the 2nd item in https://github.com/GoogleContainerTools/kpt/issues/1825?